scripts/deps-ppa/static_z3.sh

```diff
@@ -76,7 +76,7 @@ Build-Depends: debhelper (>= 9.0.0),
                git,
                libgmp-dev,
                dh-python,
-               python
```

Comment:
This script is not run by the CI, we have to try creating a ppa package locally.

Reply:
/cc @ekpyron  - I may need your advice for that :)

Reply:
I can check - but I'd also say it'd be fine to leave this script as is for now and only update it the next time we need to run it...